omero/sysadmins/server-upgrade.txt

```diff
@@ -73,6 +73,11 @@ image after upgrade will be slower. After re-initialization, a new memoization
 file will be automatically generated and OMERO will be able to load images in
 a performant manner again.
 
+These files are stored under :file:`BioFormatsCache` in the OMERO data
+directory, e.g. :file:`/OMERO/BioFormatsCache`. You may see error messages in
+your log files when an old memoization file is found, to avoid these messages
```

Comment:
Last minor nit is this should be a semicolon or new sentence rather than a comma.